From b25518e13179bc6553bd8b0d9a0f554bcf7bd288 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Sep 2014 00:20:34 +0100
Subject: [PATCH 2/2] autotools: Use summaryInWorkdir()

.. for the configure step. Though IMHO displaying the relative
path would look better.

Change-Id: If5ec2597d8c4ae064281f848104a8eb99f2c5d1f
---
 src/plugins/autotoolsprojectmanager/configurestep.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/autotoolsprojectmanager/configurestep.cpp b/src/plugins/autotoolsprojectmanager/configurestep.cpp
index 06acfb9..f97f094 100644
--- a/src/plugins/autotoolsprojectmanager/configurestep.cpp
+++ b/src/plugins/autotoolsprojectmanager/configurestep.cpp
@@ -298,6 +298,6 @@ void ConfigureStepConfigWidget::updateDetails()
     param.setWorkingDirectory(bc->buildDirectory().toString());
     param.setCommand(projectDirRelativeToBuildDir(bc) + QLatin1String("configure"));
     param.setArguments(m_configureStep->additionalArguments());
-    m_summaryText = param.summary(displayName());
+    m_summaryText = param.summaryInWorkdir(displayName());
     emit updateSummary();
 }
-- 
2.1.0

